DSA Session/Assignment2/CircularQueueUsingArray/src/queue/QueueImpl.java

```diff
@@ -0,0 +1,81 @@
+package queue;
+
+public class QueueImpl implements Queue {
+
+    int[] elements;
+    int front;
+    int rear;
+    int maxSize;
+    
+    public QueueImpl(int length) {
```

Comment:
Where is the documentation for this method?

Reply:
I missed it, I will do it.